src/Installers/Windows/Wix.targets

```diff
@@ -64,18 +64,12 @@
 
   <Target Name="CopyToArtifactsDirectory"
           Condition=" '$(IsShipping)' == 'true' AND '$(SkipCopyToArtifactsDirectory)' != 'true' "
-          AfterTargets="Build" BeforeTargets="GenerateChecksums">
+          AfterTargets="Build">
```

Comment:
Use `BeforeTargets="Build"` because `Build` is a no-op target that comes at the _end_ of the build process. As-is, there's no guarantee when this target will run. (Actually, it'll likely run when the calling project is about to exit and that's long before AfterSigning.targets would execute. Still good practice.)

Reply:
This is an undo of a change I made in my first PR to generate checksums - it's intended to no-op to where we were before I added checksum stuff to the repo. This project should now do nothing checksum-related. See:

https://github.com/dotnet/aspnetcore/blob/03b4bcfee8a2e8de292e3bbfceaca9d58b60e92e/src/Installers/Windows/Wix.targets#L65-L75

Reply:
That's fine but `BeforeTargets` pointing to a no-op target (like `Build`) is _always_ more deterministic and reliable than any `AfterTargets`